src/components/Sherlock/Sherlock.tsx

```diff
@@ -35,12 +35,15 @@ const Sherlock = () => {
     const description =
         "Sherlock is a tool used for searching social networks for specified usernames.\n\n" +
         "Further information can be found at: https://www.kali.org/tools/sherlock/\n\n" +
-        "Using Sherlock:\n\n" +
+        "Using Sherlock:\n\n";
+
+    const steps =
         "*Note: For multiple usernames, add a space in between. E.g. 'Greg John Billy'*\n\n" +
         "Step 1: Input the username(s) you wish to search for in the Username field.\n" +
         "       Eg: socialperson38\n\n" +
         "Step 2: Click Start Searching to commence Sherlock's operation.\n\n" +
         "Step 3: View the Output block below to view the results of the tool's execution.";
+
```

Comment:
```suggestion
    const steps =
        "Using Sherlock:\n\n";
        "*Note: For multiple usernames, add a space in between. E.g. 'Greg John Billy'*\n\n" +
        "Step 1: Input the username(s) you wish to search for in the Username field.\n" +
        "       Eg: socialperson38\n\n" +
        "Step 2: Click Start Searching to commence Sherlock's operation.\n\n" +
        "Step 3: View the Output block below to view the results of the tool's execution.";
```
I suggest moving this to the steps as well, make sure to run prettier again to pass style check.